### PR TITLE
Add docker image containing nodejs auto instrumentation

### PR DESCRIPTION
**Pull request by anuraaga**
Created image: https://github.com/anuraaga/opentelemetry-operator/pkgs/container/opentelemetry-operator%2Fautoinstrumentation-nodejs

I think the autoinstrumentation wrapper could be pushed to js-contrib though we'll still need to package something with Docker. Hopefully it's OK to include it here for now to try out the autoinstrumentation experience.

@pavolloffay @jpkrohling 

/cc @dyladan 

Would be used by https://github.com/open-telemetry/opentelemetry-operator/pull/507

### CODE REVIEW

File: autoinstrumentation/nodejs/src/autoinstrumentation.ts
Comment (anuraaga) [inline autoinstrumentation/nodejs/src/autoinstrumentation.ts:2]: From what I understand NodeJS doesn't support env vars like OTEL_TRACES_EXPORTER yet. When it does, we would remove this sort of hard-coding.

Comment (jpkrohling) [reply autoinstrumentation/nodejs/src/autoinstrumentation.ts:2]: Should this be an issue on this repository then?

**Review by jpkrohling — COMMENTED** [review]
I have no idea about the nodejs parts, but the github workflow is looking good. I will wait for a review from someone who knows about the nodejs parts before merging.

**Comment by pavolloffay** [comment]
@anuraaga how does the injection of nodeJS instrumentation to an application work? Could you please briefly describe here what the operator will have to do?

**Comment by anuraaga** [comment]
@pavolloffay The code is in #507 :) But if it would help to also have text explanation let me know

**Review by pavolloffay — APPROVED** [review]

**Review by jpkrohling — APPROVED** [review]

**Pull request merged**
